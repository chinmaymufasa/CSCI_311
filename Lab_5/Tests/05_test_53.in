2
6
4
2
123
5
2
105
6
2
96
5
2
96
6
-1
