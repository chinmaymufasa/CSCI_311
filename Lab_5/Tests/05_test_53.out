6 
6 123 
6 123 105 
6 96 105 123 
6 96 96 123 105 
